src/components/staking/ValidatorIcon.vue

```diff
@@ -15,6 +16,14 @@ export default defineComponent({
             required: true,
         },
     },
+    setup(props) {
+        const { config } = useConfig();
+
+        return {
+            src: 'logoPath' in props.validator
+                ? `${config.staking.validatorsEndpoint}}/${props.validator.logoPath}` : undefined,
```

Comment:
Is this deployed to validator-api yet? Or should we keep the fallback of `logo`?

Reply:
This is not deployed yet